core/src/test/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -1672,6 +1696,25 @@ protected Settings nodeSettings(int nodeOrdinal) {
     }
 
     /**
+     * Returns a collection of plugins that should be loaded on each node.
+     */
+    protected Collection<Class<? extends Plugin>> nodePlugins() {
+        return Collections.emptyList();
+    }
+
+    /**
+     * Returns a collection of plugins that should be loaded when creating a transport client.
+     */
+    protected Collection<Class<? extends Plugin>> transportClientPlugins() {
+        return Collections.emptyList();
+    }
+
+    /** Helper method to create list of plugins without specifying generic types. */
+    protected static Collection<Class<? extends Plugin>> pluginList(Class<? extends Plugin>... plugins) {
```

Comment:
This method should have the `@SafeVarargs` annotation, otherwise every use of it will get a warning.